src/components/TransactionStatus.svelte

```diff
@@ -8,7 +8,7 @@
   };
 </script>
 
-<div class:hidden={!showSelf} id="transaction-status" class="action-card grow-1 ml-8 basis-1/3">
+<div class:hidden={!showSelf} id="transaction-status" class="">
```

Comment:
Text is so large in transaction status. Consider minor styling or creating a new issue for styling of this.

Reply:
https://github.com/LibertyDSNP/provider-dashboard/issues/143